docs/REST API/rest-api.md

```diff
@@ -3663,6 +3665,114 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ```
 
+### Get host's software
+
+`GET /api/v1/fleet/hosts/:id/software`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| id   | integer | path | **Required**. The host's ID. |
+| page | integer | query | Page number of the results to fetch.|
+| per_page | integer | query | Results per page.|
+
+#### Example
+
+`GET /api/v1/fleet/hosts/123/software`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "software": [
+    {
+      "id": 121,
+      "name": "Google Chrome.app",
+      "package_available_for_install": "GoogleChrome.pkg",
+      "source": "apps",
+      "bundle_identifier": "com.google.Chrome",
+      "status": "failed",
+      "detail": "Software is installed, but script after installation failed.",
+      "last_install": {
+        "install_uuid": "8bbb8ac2-b254-4387-8cba-4d8a0407368b",
+        "installed_at": "2024-05-15T15:23:57Z"
+      },
+      "installed_versions": [
+        { 
+          "version": "121.0",
+          "last_opened_at": "2024-04-01T23:03:07Z",
+          "vulnerabilities": ["CVE-2023-1234","CVE-2023-4321","CVE-2023-7654"],
+          "installed_paths": ["/Applications/Google Chrome.app"]
+        }
+      ]
+    },
+    {
+      "id": 134,
+      "name": "Falcon.app",
+      "software_package": "FalconSensor-6.44.pkg",
+      "source": "",
+      "bundle_identifier": "",
+      "status": null,
+      "detail": "",
+      "last_install": null,
+      "installed_versions": [],
+    },
+    {
+      "id": 147,
+      "name": "Firefox.app",
+      "software_package": null,
+      "source": "apps",
+      "bundle_identifier": "org.mozilla.firefox",
+      "status": null,
+      "detail": "",
+      "last_install": null,
+      "installed_versions": [
+        {
+          "version": "118.0",
+          "last_opened_at": "2024-04-01T23:03:07Z",
+          "vulnerabilities": ["CVE-2023-1234"],
+          "installed_paths": ["/Applications/Firefox.app"]
+        },
+        { 
+          "version": "119.0",
+          "last_opened_at": "2024-04-01T23:03:07Z",
+          "vulnerabilities": ["CVE-2023-4321","CVE-2023-7654"],
+          "installed_paths": ["/Downloads/Firefox.app"]
+        }
+      ]
+    },
+  ],
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
+}
+```
+
+### Install software
```

Comment:
@marko-lisica I think this (and Add/Download/Delete software below, and Get installation result below) should mention that they are premium-only?

Reply:
@mna Thanks for catching this. Added `Available in Fleet Premium` for each of them.